src/main/java/ch/wisv/areafiftylan/controller/TicketRestController.java

```diff
@@ -25,11 +27,13 @@
 public class TicketRestController {
     private TicketService ticketService;
     private OrderService orderService;
+    private RFIDService rfidService;
 
     @Autowired
-    public TicketRestController(TicketService ticketService, OrderService orderService) {
+    public TicketRestController(TicketService ticketService, OrderService orderService, RFIDService rfidService) {
```

Comment:
No RFID stuff in the TicketController please. You can move these to endpoints in the RFIDController such as `/rfid/tickets`